### ISSUE TITLE
Add search filters

### ISSUE THREAD

**Issue by galantra** [ll-in-anki/anki-LL#8]
This is a great idea and implementation.
Something I'd add: The possibility to apply same filters that can be used in Anki's search and filtered decks, for example:
`-is:suspended
added:7
tag:English
`

**Comment by jeprecated** [comment]
There was a manual entry box much earlier on in the project, where any query could be entered just like in the browser and added would be to the calculated query. We mainly used this for debugging purposes, but I can see the benefit here for advanced users. It would be very simple to add again 👍 

Proper UI for all the filter options in Anki is probably a longer way off, but will probably make it's way in at some point 😁

**Comment by kennethbruskiewicz** [comment]
I agree with this feature. In my case I want to apply this to dense technical texts written in English. I would like to exclude common words/stop words when assessing these texts.

**Comment by cofinley** [comment]
@kbruskiewicz, while we will still look into extra filtering, but in case you didn't see, if you want to exclude words, you can do that from the config (or after clicking on "Ignore" in the search results). Good for stop words, articles, proper nouns, stuff you just know by heart, etc.

Ignoring en masse in the config:

![](https://camo.githubusercontent.com/bc761ad15aa0238a7374ecc12a5f643b4b7ea954/68747470733a2f2f692e696d6775722e636f6d2f366d6c793171752e706e67)

Ignoring by word (see top right):

![](https://camo.githubusercontent.com/f3a6175dc703e69de61a8bd2f4c215197b176f06/68747470733a2f2f692e696d6775722e636f6d2f386c4c474a30322e706e67)
